documentation/src/main/markdown/releasenotes.md

```diff
@@ -14,6 +14,7 @@
 ### Resolved issues
 
 * (IDETECT-3111) When scanning SBT projects, "Evicted" dependencies are appearing in the resulting BOM when they should be excluded.
+* (IDETECT-3685) Null Pointer Exception when parsing a Podfile.lock file without PODS.
```

Comment:
@gopannabd Can we say something like "gracefully handled use case when no components are found parsing a Poffile.lock file without PODS instead of failing with NPE? and not add just add the ticket description! cc: @sig-cpotts. 

Not related to this one, but the above item IDETECT-3111 note should also say what we fixed rather than just describing the problem.

Reply:
I have updated both!

Reply:
@dmamidisynopsys Note that for IDETECT-3111 the "Resolved" issue note described the issue, and the Changed feature section described the new behavior related to the fix.